apps/opik-backend/src/main/java/com/comet/opik/domain/ExperimentItemDAO.java

```diff
@@ -260,4 +269,27 @@ private Publisher<? extends Result> delete(Set<UUID> ids, Connection connection)
 
         return makeFluxContextAware(bindWorkspaceIdToFlux(statement));
     }
+
+    public Mono<Long> deleteByExperimentId(UUID id) {
+        Preconditions.checkArgument(Objects.nonNull(id), "Argument 'id' must not be null");
```

Comment:
Same.

Reply:
Same

Reply:
Correct. After changing to a set, the checkArgument without NonNull annotation is preferable.